recipes/fs.sshfs/meta.yaml

```diff
@@ -0,0 +1,125 @@
+{% set name = "fs.sshfs" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e9f218a382c8a82c9f639d1c0e3a622ed005fed03923966621bafbfe0da12f4
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - skip: true  # [win]
```

Comment:
This is in the wrong place and may be the culprit of the failure you are seeing.

Reply:
seems I am blind. Thx, lets see.